wavefront_opentracing_python_sdk/tracer.py

```diff
@@ -0,0 +1,207 @@
+"""
+Wavefront Tracer.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+
+import time
+import uuid
+from opentracing import Tracer, Reference, ReferenceType, \
+    UnsupportedFormatException
+from opentracing.scope_managers import ThreadLocalScopeManager
+from wavefront_opentracing_python_sdk.propagation import registry
+from wavefront_opentracing_python_sdk import WavefrontSpan, \
```

Comment:
`WavefrontSpan` lives in ` wavefront_opentracing_python_sdk.span` and `WavefrontSpanContext` lives in `wavefront_opentracing_python_sdk.span_context`